public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -92,8 +93,35 @@ export class PanelHeader extends Component<Props, State> {
     );
   }
 
+  openInspect = (e: React.SyntheticEvent, tab: string) => {
+    const { updateLocation, panel } = this.props;
+
+    e.stopPropagation();
+
+    updateLocation({
+      query: { inspect: panel.id, tab },
+      partial: true,
+    });
+  };
+
+  renderNotice = (notice: QueryResultMetaNotice) => {
+    return (
+      <Tooltip content={notice.text} key={notice.severity}>
+        {notice.inspect ? (
+          <div className="panel-info-notice" onClick={e => this.openInspect(e, notice.inspect)}>
+            <span className="fa fa-info-circle" style={{ marginRight: '8px', cursor: 'pointer' }} />
+          </div>
+        ) : (
+          <a className="panel-info-notice" href={notice.url} target="_blank">
+            <span className="fa fa-info-circle" style={{ marginRight: '8px', cursor: 'pointer' }} />
+          </a>
+        )}
+      </Tooltip>
+    );
```

Comment:
will revisit this part in later PRs, try to unify this a bit more with panel description, links & alert heart, and panel errors